src/Microsoft.Data.Analysis/DataFrameColumn.cs

```diff
@@ -24,6 +24,45 @@ public DataFrameColumn(string name, long length, Type type)
             DataType = type;
         }
 
+        /// <summary>
+        /// A static factory method to create a <see cref="PrimitiveDataFrameColumn{T}"/>. 
+        /// It allows you to take advantage of type inference based on the type of the values supplied.
+        /// </summary>
+        /// <typeparam name="T">The type of the column to create.</typeparam>
+        /// <param name="name">The name of the column.</param>
+        /// <param name="values">The initial values to populate in the column.</param>
+        /// <returns>A <see cref="PrimitiveDataFrameColumn{T}"/> populated with the provided data.</returns>
+        public static PrimitiveDataFrameColumn<T> Create<T>(string name, IEnumerable<T?> values) where T : unmanaged
+        {
+            return new PrimitiveDataFrameColumn<T>(name, values);
+        }
+
+        /// <summary>
+        /// A static factory method to create a <see cref="PrimitiveDataFrameColumn{T}"/>. 
+        /// It allows you to take advantage of type inference based on the type of the values supplied.
+        /// </summary>
+        /// <typeparam name="T">The type of the column to create.</typeparam>
+        /// <param name="name">The name of the column.</param>
+        /// <param name="values">The initial values to populate in the column.</param>
+        /// <returns>A <see cref="PrimitiveDataFrameColumn{T}"/> populated with the provided data.</returns>
+        public static PrimitiveDataFrameColumn<T> Create<T>(string name, IEnumerable<T> values) where T : unmanaged
+        {
+            return new PrimitiveDataFrameColumn<T>(name, values);
+        }
+
+        /// <summary>
+        /// A static factory method to create a <see cref="StringDataFrameColumn{T}"/>. 
+        /// It allows you to take advantage of type inference based on the type of the values supplied.
+        /// </summary>
+        /// <typeparam name="T">The type of the column to create.</typeparam>
```

Comment:
There is no `T` type param.

These XML errors should have given you at least a warning. Are you seeing any?

Reply:
Sorry I missed this message- no I didn't see any warnings. I've removed the errant `{T}`.

Reply:
I think you missed the `T` in the documentation comments on lines 57 and 60?